src/app.js

```diff
@@ -23,6 +25,18 @@ app.use(
 app.use(passport.initialize());
 app.use(passport.session());
 app.use('/auth', authRouter);
+const { newsletter, serverStatus } = require('./utils/cron');
+
+const allProjects = async () => {
+  const projects = await Project.find();
+  const titles = [];
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < projects.length; i++) {
+    titles.push(projects[i].title);
+  }
```

Comment:
Instead of sending all projects in the db, you can send only the the projects created in a specific time frame.